core/src/com/unciv/logic/automation/NextTurnAutomation.kt

```diff
@@ -19,27 +19,32 @@ class NextTurnAutomation{
 
     /** Top-level AI turn tasklist */
     fun automateCivMoves(civInfo: CivilizationInfo) {
-        respondToDemands(civInfo)
-        respondToTradeRequests(civInfo)
-
-        if(civInfo.isMajorCiv()) {
-            offerPeaceTreaty(civInfo)
-            exchangeTechs(civInfo)
-            exchangeLuxuries(civInfo)
-            issueRequests(civInfo)
-            adoptPolicy(civInfo)
+        if (civInfo.isBarbarian()) {
+            BarbarianAutomation(civInfo).automate()
```

Comment:
We could just return here, and then the second but won't need to be in an "else"

Reply:
Same as last time, I'll deal with it, but it's something you need to pay attention to.
It leaves the left side of the function as the "happy path", and it means that people who read the function don't need to check what their scope because there's no indentation, so they're on the "regular path"